### DISCUSSION TITLE
Completely wrong energy flow dashboard

### DISCUSSION THREAD

**Discussion by MG-Programmer** [wlcrs/huawei_solar#381] [General]
Hello,

I have configured this integration to visualize my inverters and battery.
I can read all inverter, battery and meter data with their correct values in the integration/devices area.
But when I want to configure the energy dashboard the energy flow chart is completely wrong.
Can someone please help me to configure it correct so it shows nearly the same as in the SolarFusion App?

![SuloarFusion energy flow](https://github.com/wlcrs/huawei_solar/assets/27810898/2b0f330b-073d-4aac-91cf-bd9d5d1a0c9b)
![Home Assistant energy flow](https://github.com/wlcrs/huawei_solar/assets/27810898/563d3674-9879-4cbc-9de4-31788c402c83)
![Energy config](https://github.com/wlcrs/huawei_solar/assets/27810898/ab63dedc-8a91-4d70-90c5-b8c2ec786995)

**Comment by jires** [comment]
Why do you think it is wrong? On the picture from FusionSolar you have a power (kW) at the very moment. In energy dashboard is shown energy (kWh) at time interval (day, week, month, etc)
